patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+InnerSource Hackathon
+
+## Patlet
+
+In a company, initially only InnerSource enthusiasts are interested and practicing InnerSource during the early stages of InnerSource adoption; not all engineering teams are willing or have enough time and resources to adopt InnerSource. In this scenario, it is good to provide a safe space to try and adopt InnerSource through an InnerSource Hackathon event within the company.
+
+## Problem
+
+The company wants to adopt InnerSource as software development methodology but only those familiar with open source principles or those who understand the benefits of InnerSource, adopt it. It results in just a handful of InnerSource projects and is difficult to scale beyond that.
+
+## Context
+
+Scenario 1:
```

Comment:
If you were to give these scenarios names, how would you call them?
What I am after is the essence of each scenario.

Reply:
I would say - 
Scenario 1: Challenges in scaling beyond the early adopters
Scenario 2: Challenges in getting contributions and building community around InnerSource projects

Reply:
I have worked these proposals into subsection headings. 👍